ocfweb/docs/docs/staff/mailing-lists.md

```diff
@@ -0,0 +1,53 @@
+[[!meta title="Staff mailing lists"]]
+
+## Mailing lists
+
+There are several mailing lists used internally by staff. Append
+`@ocf.berkeley.edu` to the end of each mailing list.
+
+All staffers are automatically added to the following two mailing lists:
+
+ * `staff`: For staff announcements, such as meeting times and events.
+ * `wheel`: For technical discussion among all staff
+
+Staffers can also choose to be further added to the following mailing lists.
+Technical Managers are required to join them:
+
+ * `rt`: Emails sent to Request Tracker are copied to this mailing list. If
+   you are on the `rt` mailing list, you can reply to RT tickets from your
+   email. You are highly encouraged to join this mailing list even if you're
+   not root staff.
+ * `root`: Miscellaneous messages from system daemons are sent here:
+    * Cron daemons send mail containing any stdout/stderr output from cronjobs
+    * Jenkins sends emails whenever a Jenkins build fails
+    * ocflib sends emails whenever an uncaught exception is thrown in ocfweb,
+      create, enforcer, and several other background tasks
+    * Miscellaneous other emails are sent here
+   This mailing list gets a median of ~10 messages every day, although on some
+   days it can get a lot more.
+ * `puppet`: Error messages from puppet runs go here. This list tends to be
+   very noisy.
+ * `mon`: Monitoring alerts are sent here:
+    * Rackspace Cloud Monitoring email us alerts when our important services are
```

Comment:
Maybe link the to [monitoring login page](https://intelligence.rackspace.com/login) or something? I don't think there's a link anywhere else that I know of.

Reply:
Also I think this should be `emails` instead of `email`?

Reply:
Sure, we could add a link, although if I were trying to find the Rackspace login link, I wouldn't think of looking on this page first...

Reply:
Yeah, that's fair, I suppose we should put a link somewhere else then, I just figured if it was being mentioned that it would be useful to have a link to it as well for more information if wanted.